virtualmachine/vsphere/vm.go

```diff
@@ -1463,6 +1479,16 @@ func getVirtualMachines(vm *VM) ([]mo.VirtualMachine, error) {
 		return nil, err
 	}
 
+	if vm.Destination.DestinationName == "" {
+		// Return templates for the whole datacenter
+		dcObj, err := vm.finder.Datacenter(vm.ctx, vm.Datacenter)
```

Comment:
I tried using it at first, but it returns *mo.Datacenter, which does not match the arg type of `getDcVMList()`